### ISSUE TITLE
Can not get 51.8mm MPJPE when retrain a model on Human3.6M in fully supersived setting

### ISSUE THREAD

**Issue by wanghonghui1998** [mkocabas/EpipolarPose#36]
Hi, thank you for sharing the codes! 
I retrained the model in fully supervised setting and got 54.6, 54.9, 55.3, 56.2(hm36_17j). And I valid the best model you provided and got 52.9 which should be 51.8.
Could you help me find out where the problem is? Here is my configuration:
Python 3.7.1
Pytorch 1.3.1
Cuda 10.0.130

Besides, I found some annotation error in S9_SittingDown_15011271, S9_SittingDown_158860488, S9_SittingDown_160457274, 75 images in all. I do not go over all the images, so I do not know if there are other wrong annotations. I think that maybe a reason for the lower performance.

**Comment by wanghonghui1998** [comment]
Here is the wrong annotation example
<img width="348" alt="image-20200106224006686" src="https://user-images.githubusercontent.com/48725183/71875258-da74dc80-315e-11ea-8c00-53fdac0c9ca8.png">

**Comment by kyang-06** [comment]
Hi, I met the same reproducing issue. Under default training setting, I only got MPJPE=55.6mm.

But the disalignment problem you mentioned is not the reason. MPJPE measures the distance all joints away from the root joint, so the global offset is subtracted in such process. 
More discussion about disalignment refers here [Learnable triangulation](https://saic-violet.github.io/learnable-triangulation/)

> Here is the wrong annotation example
> <img alt="image-20200106224006686" width="348" src="https://user-images.githubusercontent.com/48725183/71875258-da74dc80-315e-11ea-8c00-53fdac0c9ca8.png">

**Comment by youpJiang** [comment]
> Here is the wrong annotation example <img alt="image-20200106224006686" width="348" src="https://user-images.githubusercontent.com/48725183/71875258-da74dc80-315e-11ea-8c00-53fdac0c9ca8.png">

Did u figure out the problem now? I met the same problem but worse.

**Comment by kyang-06** [comment]
> > Here is the wrong annotation example <img alt="image-20200106224006686" width="348" src="https://user-images.githubusercontent.com/48725183/71875258-da74dc80-315e-11ea-8c00-53fdac0c9ca8.png">
> 
> Did u figure out the problem now? I met the same problem but worse.

The problem comes from error annotation of h36m, which is raised by the work [Learnable Triangulation of Human Pose](https://saic-violet.github.io/learnable-triangulation) at Part Human3.6M erroneous annotations.

One solution is to directly remove these errornous video clips from test set.

**Comment by youpJiang** [comment]
> 

Thanks for your reply! 
I valid the best model author provided and got 52.9 as same as the issue author saied,but only got 62\64 (hm36_17j) when retrained the model in fully supervised setting(with MPII pretrained model). 

Will there be some errors in the training set?Could u give me some advice,please？

**Comment by youpJiang** [comment]
> > > Here is the wrong annotation example <img alt="image-20200106224006686" width="348" src="https://user-images.githubusercontent.com/48725183/71875258-da74dc80-315e-11ea-8c00-53fdac0c9ca8.png">
> > 
> > 
> > Did u figure out the problem now? I met the same problem but worse.
> 
> The problem comes from error annotation of h36m, which is raised by the work [Learnable Triangulation of Human Pose](https://saic-violet.github.io/learnable-triangulation) at Part Human3.6M erroneous annotations.
> 
> One solution is to directly remove these errornous video clips from test set.

Thanks for your reply!
I valid the best model author provided and got 52.9 as same as the issue author said,but only got 62\64 (hm36_17j) when retrained the model in fully supervised setting(with MPII pretrained model).

Will there be some errors in the training set?Could u give me some advice,please？

**Comment by kyang-06** [comment]
@youpJiang I'm not sure whether the drifting error exists in training set, probably none as far as i know.
Maybe you can have a try with more accurate 2D detector, such as HRNet refined on h36m as provided by [https://github.com/Nicholasli1995/EvoSkeleton](https://github.com/Nicholasli1995/EvoSkeleton).

**Comment by youpJiang** [comment]
@kyang-06 Thanks for your advice! Wish u academic success.

**Comment by kyang-06** [comment]
@youpJiang you are welcome, wish you good result too.
